cmd/server/main.go

```diff
@@ -165,6 +165,8 @@ func main() {
 		Repo: accountsRepo, WatchmanClient: watchmanClient,
 	}
 
+	bucket := storage.GetBucket(logger, os.Getenv("DOCUMENTS_BUCKET"), util.Or(os.Getenv("DOCUMENTS_PROVIDER"), "file"), signer)
```

Comment:
https://github.com/peterbourgon/ff would useful here for setting config data. 

From highest to lowest priority, it'll check:
1. CLI flags
2. env vars
3. default value

Reply:
Long term I want a configuration file like PayGate. Those are a lot easier to use than all these env vars, but that change doesn't fit in here.

Reply:
Sounds good, I'll take a look at PayGate's config setup.